identity/client/src/components/layout/AppHeader.tsx

```diff
@@ -3,11 +3,19 @@ import { useGlobalRoutes } from "src/components/global/useGlobalRoutes";
 import { useNavigate } from "react-router";
 import { useApi } from "src/utility/api";
 import { checkLicense, License } from "src/utility/api/headers";
+import { getAuthentication } from "src/utility/api/authentication";
+import { useCallback } from "react";
+import { ArrowRight } from "@carbon/react/icons";
 
 const AppHeader = () => {
   const routes = useGlobalRoutes();
   const navigate = useNavigate();
   const { data: license } = useApi(checkLicense);
+  const { data: camundaUser } = useApi(getAuthentication);
+
+  const logout = useCallback(() => {
+    console.log("logout");
```

Comment:
🔧 Not sure if you intentionally left this in :D

Reply:
Yes, although if you prefer I could also remove the whole callback. I had replaced this with an actual logout action in https://github.com/camunda/camunda/pull/26262

Reply:
Ah I'm fine with either if we're going to address this proactively in a upcoming PR